atlasdb-cassandra/src/main/java/org/apache/commons/pool2/impl/ExposedEvictionTimer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.commons.pool2.impl;
+
+/**
+ * This class exists solely to expose package-private methods on {@link EvictionTimer}; this should be removed once
+ * the internal investigation into the missing eviction problem is resolved.
+ */
+public final class ExposedEvictionTimer {
+    private ExposedEvictionTimer() {
+        // no-op
+    }
+
+    public static long getNumTasks() {
+        return EvictionTimer.getNumTasks();
+    }
```

Comment:
This is pretty sneaky.

Since we're in the apache package, do we need a license?

Reply:
Perfect.